test/index.test.js

```diff
@@ -72,9 +72,9 @@ describe('Vero', function() {
         analytics.stub(window._veroq, 'push');
       });
 
-      it('should push "trackPageview"', function() {
+      it('should push "viewed_page"', function() {
         analytics.page();
-        analytics.called(window._veroq.push, ['trackPageview']);
+        analytics.called(window._veroq.push, ['track', 'viewed_page', {}, { source: 'segment' }]);
```

Comment:
Thanks for that,

I was able to fix the problem by passing in the `referrer` to `#page()`, this means the example will always have the same `id` parameter. 

Let me know if theres a better way to accomplish this!